src/main/resources/js/condec.api.js

```diff
@@ -682,6 +707,38 @@
 			}
 		});
 	};
+	
+	ConDecAPI.prototype.getDecisionGroups = function getDecisionGroups(id, location,inputExistingGroupsField,selectLevelField, callback) {
+		var projectKey = getProjectKey();
+		var decisionGroups = getResponseAsReturnValue(AJS.contextPath() + "/rest/condec/latest/config/getDecisionGroups.json?elementId=" +id
+				 + "&location="+location+ "&projectKey="+projectKey);
+		callback(selectLevelField, inputExistingGroupsField,decisionGroups);
+	};
+	
+	ConDecAPI.prototype.fillDecisionGroupSelect = function fillDecisionGroupSelect(elementId){
+		var selectGroupField = document.getElementById(elementId);
+		getAllDecisionGroups(selectGroupField,function(selectGroupField, groups){
+			if(!(groups === null) && groups.length > 0){
+				selectGroupField.innerHTML= "";
+				selectGroupField.insertAdjacentHTML("beforeend", "<option value='High_Level'>High_Level</option>"
+						+"<option value='Medium_Level'>Medium_Level</option>"
+						+"<option value='Realization_Level'>Realization_Level</option>");
+				for(var i = 0; i< groups.length; i++){
+					if(groups[i]!= "High_Level" && groups[i]!= "Medium_Level" && groups[i]!= "Realization_Level"){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5160185)